src/app/loadout/mod-utils.ts

```diff
@@ -98,3 +99,37 @@ export function createGetModRenderKey() {
 export function groupModsByModType(plugs: PluggableInventoryItemDefinition[]) {
   return _.groupBy(plugs, (plugDef) => plugDef.itemTypeDisplayName);
 }
+
+/**
+ * Some mods have two copies, a regular version and a reduced-cost version.
+ * Only some of them are seasonally available, depending on artifact mods/unlocks.
+ * This maps to whichever version is available, otherwise returns plugHash unmodified.
+ */
+export function mapToAvailableModCostVariant(plugHash: number, unlockedPlugs: Set<number>) {
+  if (unlockedPlugs.has(plugHash)) {
+    return plugHash;
+  }
```

Comment:
Should we use the lower-cost mod if it's available even if the higher-cost was explicitly in the loadout? That is, should we move this after the `toReduced` check?

Reply:
The game disables the regular version if the cheap version is available and the API reports the regular version as locked, so the Mod Picker should always only show at most one version of a mod.